src/sentry/templates/sentry/login.html

```diff
@@ -13,16 +13,24 @@ <h2 class="m-t-1 m-l-1 m-b-0">{{ banner }}</h2>
   <div class="auth-container p-t-1 border-bottom">
     <h3>{% trans "Sign in to continue" %}</h3>
     <ul class="nav nav-tabs auth-toggle m-b-0">
-      <li{% if op == "login" %} class="active"{% endif %}>
-        <a href="#login" data-toggle="tab">{% trans "Sign In" %}</a>
+       <!-- Sign In Tab -->
+      {% url 'sentry-login' as login_url %}
+      <li{% if request.path == login_url %} class="active"{% endif %}>
+        <a href="{{ login_url }}">{% trans "Single In" %}</a>
       </li>
+
+      <!-- Register Tab -->
       {% if CAN_REGISTER %}
-        <li{% if op == "register" %} class="active"{% endif %}>
-          <a href="#register" data-toggle="tab">{% trans "Register" %}</a>
+        {% url 'sentry-register' as register_url %}
+        <li{% if request.path == register_url %} class="active"{% endif %}>
+          <a href="{{ register_url }}">{% trans "Register" %}</a>
         </li>
       {% endif %}
-      <li{% if op == "sso" %} class="active"{% endif %}>
-        <a href="#sso" data-toggle="tab">{% trans "Single Sign-On" %}</a>
+
+      <!-- Single Sign-On Tab -->
+      {% url 'sentry-auth-sso' as sso_url %}
+      <li{% if request.path == sso_url %} class="active"{% endif %}>
+        <a href="{{ sso_url }}">{% trans "Single Sign-On" %}</a>
       </li>
```

Comment:
This here was broken and I took the opportunity to quickly fix it

**Before:**


https://github.com/user-attachments/assets/8d1706f9-120d-4e61-a09b-67021410f0b2




**After**


https://github.com/user-attachments/assets/ee7469fd-9e49-423a-9874-88bb12ff49c1